variation_test.go

```diff
@@ -46,7 +46,10 @@ func (c *cacheMock) GetLatestUpdateDate() time.Time {
 func (c *cacheMock) ConvertToFlagStruct(_ []byte, _ string) (map[string]dto.DTO, error) {
 	return nil, nil
 }
-func (c *cacheMock) UpdateCache(_ map[string]dto.DTO, _ *fflog.FFLogger) error {
+func (c *cacheMock) UpdateCache(_ map[string]dto.DTO, _ *fflog.FFLogger, _ bool) error {
+	return nil
+}
+func (c *cacheMock) UpdateCacheAndNotify(_ map[string]dto.DTO, _ *fflog.FFLogger) error {
 	return nil
 }
```

Comment:
✂️ This function is not part of the interface anymore, we should probably remove it from the mock.